tests/tensor/test_math_opt.py

```diff
@@ -3596,8 +3596,8 @@ def test_local_reduce_broadcast_some_0(self):
             at_all,
             at_any,
             prod,
-            at_max,
-            at_min,
+            # at_max,
+            # at_min,
```

Comment:
Why have the `max` and `min` functions been removed from `TestLocalReduce`?

Reply:
@brandonwillard I moved the `MaxAndArgmax` functionality to `Max` and now `Max` is implemented as a `COp`, so the tests for `CAReduce` will not work with `Max`.